packages/meditrak-server/src/database/models/UserEntityPermission.js

```diff
@@ -11,6 +11,28 @@ export class UserEntityPermissionModel extends CommonUserEntityPermissionModel {
   notifiers = [onUpsertSendPermissionGrantEmail];
 }
 
+const EMAILS = {
+  tupaia: {
+    subject: 'Tupaia Permission Granted',
+    body: (userName, permissionGroupName, entityName) =>
+      `Hi ${userName},\n\n` +
+      `This is just to let you know that you've been added to the ${permissionGroupName} access group for ${entityName}. ` +
+      'This allows you to collect surveys through the Tupaia data collection app, and to see reports and map overlays on Tupaia.org.\n\n' +
+      "Please note that you'll need to log out and then log back in to get access to the new permissions.\n\n" +
+      'Have fun exploring Tupaia, and feel free to get in touch if you have any questions.\n',
+  },
+  lesmis: {
+    subject: 'LESMIS Permission Granted',
+    body: (userName, permissionGroupName, entityName) =>
+      `Hi ${userName},\n\n` +
+      `This is just to let you know that you've been added to the ${permissionGroupName} access group for ${entityName}. ` +
+      'This allows you to see reports and map overlays on lesmis.la.\n\n' +
+      "Please note that you'll need to log out and then log back in to get access to the new permissions.\n\n" +
+      'Feel free to get in touch if you have any questions.\n',
+    signOff: 'Best regards,\nThe LESMIS Team',
```

Comment:
I like that the LESMIS team is more formal 😀

Reply:
Haha I'm not really sure why I did that but it seemed like I needed to